enterprise/api/v1/decorators.py

```diff
@@ -57,14 +61,30 @@ def _wrapped_view(request, *args, **kwargs):
             user = request.user
             pk = fn(request, **kwargs) if fn else kwargs.get('pk')
 
+            LOGGER.info(
```

Comment:
How about adding a unique keyword at the start so that all the logs related to permissions can be separated immediately? e.g 
LOGGER.info(f"[User_Permissions_Check]User {user.username} has permission: {has_permission}")

Reply:
@zamanafzal keyword added.